lib/mdot/token.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module MDOT
+  class Token < Common::RedisStore
+    redis_store REDIS_CONFIG[:mdot][:namespace]
+    redis_ttl REDIS_CONFIG[:mdot][:each_ttl]
+    redis_key :uuid
+
```

Comment:
May want to add validation for the uuid "validates :uuid, presence: true"

Reply:
good suggestion - added